tests/unit/local/lambdafn/test_zip.py

```diff
@@ -104,7 +104,7 @@ def test_successfully_unzip_from_uri(self, open_patch, requests_patch, progressb
 
         unzip_from_uri('uri', 'layer_zip_path', 'output_zip_dir', 'layer_arn')
 
-        requests_patch.get.assert_called_with('uri', stream=True)
+        requests_patch.get.assert_called_with('uri', stream=True, verify=True)
```

Comment:
Can you mock out `os.environ` here to return an empty dict or a dict without the `AWS_CA_BUNDLE` key? This will keep the tests from failing on systems that have that env var already set.

Reply:
sounds good to me.  I'll get it done.